src/stores/ToastStore.ts

```diff
@@ -68,6 +68,11 @@ export default class ToastStore extends EventEmitter {
     }
 
     dismissToast(key) {
+        if (!this.toasts.some(t => t.key === key)) {
+            console.warn(`No toast for ${key} found - cannot dismiss. Check your toastKey`);
```

Comment:
move into an else on line 88 plz, no point doing two array scans

Reply:
also I think this warning will be SUPER noisy.

Reply:
why would it be noisy? Surely we shouldn't be hiding toasts that often...

Reply:
every sync the mau toasts will try to hide
every update poll where there is no update the update toast will try to hide
etc

Reply:
we don't keep state of whether a toast is open in a lot of places, so we don't have a falling edge to call hide on

Reply:
noisy not as in often but as in very low signal to noise. Most of the times this warns will be very expected.

Reply:
oh fun. I'll just revert the commit then and we'll have to be careful.